### PR TITLE
[MAJOR][EDA-1640] change docker compose file

### PR DESCRIPTION
**Pull request by mmerida-eb**
This commit adds wumpus in docker-compose of the Edagames project. So remove quoridor and change the environment variable to expose 5501 port and listen to wumpus

### CODE REVIEW

**Review by emari-eb — APPROVED** [review]

File: docker-compose.yml [hunk 1]
```diff
@@ -58,12 +58,13 @@ services:
             - TOKEN_KEY=${SECRET_KEY_JWT}
             - WEB_SERVER_URL=django
             - WEB_SERVER_PORT=8000
-            - QUORIDOR_HOST_PORT=quoridor:50051
+            - WUMPUS_HOST_PORT=wumpus:50051
             - REDIS_HOST=redis
```
Comment (lolivera-eb) [inline docker-compose.yml]: so it will not be possible to play quoridor? i mean the idea is that both game should be available

**Review by lolivera-eb — CHANGES_REQUESTED** [review]
I want to know why the decision to take out quoridor and not use another port?

File: docker-compose.yml (hunk 1 above)
Comment (mmerida-eb) [reply docker-compose.yml]: At this moment. The idea of this sprint is to play only wumpus with actual infrastructure. When the game is tested and working. In future sprints we have the idea to implement a change where you can select between different games

**Comment by mmerida-eb** [comment]
> I want to know why the decision to take out quoridor and not use another port?

- We

**Comment by mmerida-eb** [comment]
We changed the docker-compose file. Now server expose two ports 50051 for quoridor and 50052 for wumpus.

**Comment by lolivera-eb** [comment]
as long as you continue to remove [quoridor](https://github.com/evbeda/edagames-server/pull/139#discussion_r980429670) in server in which it implies that you can not play I will not accept your PR

**Comment by mmerida-eb** [comment]
As mentioned above. The goal of this sprint is to be able to play Wumpus with the current infrastructure. We need can play to test the wumpus game.  Currently, the creation of an interface to be able to play both games at the same time surpasses the objective of the sprint. That's why we decided to change the default game on the server. In the future, the objective will be to create the interface to play quoridor or wumpus.

**Comment by mmerida-eb** [comment]
I saw my PR and make a mistake to modify the quoridor host port. Now both ports are up one in 50051 and another in 50052.

```
 - QUORIDOR_HOST_PORT=quoridor:50051
 - WUMPUS_HOST_PORT=wumpus:50052
```

In this ticket modify the server to play any games at the same time. 
[https://eventbrite.atlassian.net/browse/EDA-1655](https://eventbrite.atlassian.net/browse/EDA-1655)


To be able to play any of the two games. In this ticket the refactor of the server will be handled by changing the variables
DEFAULT_GAME
and
DEFAUL_PORT
[https://eventbrite.atlassian.net/browse/EDA-1657](https://eventbrite.atlassian.net/browse/EDA-1657)
The server through this change could play any of the two current games. Not at the same time but it will be able to accept both games

**Review by lolivera-eb — APPROVED** [review]

**Pull request merged**